tsconfig.node.json

```diff
@@ -0,0 +1,39 @@
+{
+  "compilerOptions": {
+    "tsBuildInfoFile": "./node_modules/.tmp/tsconfig.node.tsbuildinfo",
+    "target": "ES2022",
+    "lib": ["ES2023"],
+    "module": "ESNext",
```

Comment:
Remind me why we're using he capitalization "ESNext" instead of "esnext"?

Reply:
To `tsc`, the casing should not matter; however, it's `ESNext` in https://json.schemastore.org/tsconfig.json and PyCharm uses the schema file to validate any `tsconfig.json` files. `eslint` would also pass validation but auto-complete only shows `ESNext`.